guides/doc-Provisioning_Guide/topics/Cloud-Amazon_EC2.adoc

```diff
@@ -1,25 +1,38 @@
 [[Provisioning_Cloud_Instances_in_Amazon_EC2]]
 == Provisioning Cloud Instances in Amazon EC2
 
-Amazon Elastic Compute Cloud (Amazon EC2) is a web service that provides public cloud compute resources. Using {ProjectNameX}, you can interact with Amazon EC2's public API to create cloud instances and control their power management states. Use the procedures in this chapter to add a connection to an Amazon EC2 account and provision a cloud instance.
+Amazon Elastic Compute Cloud (Amazon EC2) is a web service that provides public cloud compute resources. Using {Project}, you can interact with Amazon EC2's public API to create cloud instances and control their power management states. Use the procedures in this chapter to add a connection to an Amazon EC2 account and provision a cloud instance.
 
 [[Provisioning_Cloud_Instances_in_Amazon_EC2-Prerequisites_for_Amazon_EC2_Provisioning]]
 === Prerequisites for Amazon EC2 Provisioning
 
 The requirements for Amazon EC2 provisioning include:
 
+ifeval::["{Build}" == "foreman"]
+  * The installation media that you require for the operating systems you want to use to provision using Amazon EC2.
+endif::[]
+ifeval::["{Build}" == "satellite"]
   * Synchronized content repositories for Red{nbsp}Hat Enterprise Linux 7. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red{nbsp}Hat Repositories] in the _Content Management Guide_.
+endif::[]
   * A {SmartProxyServer} managing a network in your EC2 environment. Use a Virtual Private Cloud (VPC) to ensure a secure network between the hosts and the {SmartProxyServer}.
   * An Amazon Machine Image (AMI) for image-based provisioning.
+ifeval::["{Build}" == "satellite"]
   * An activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
+ifeval::["{Build}" == "foreman"]
+  * If the Katello plugin is installed, an activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
```

Comment:
Looks probably better than my version (libvirt PR). Go ahead and compare if you like anything from my version but this is fine.

Reply:
I will read yours and then I think we should add a standardized form of this sentence to the readme so we don't rewrite it the same way every time. 
If we write it identically, this should help with machine translation, or human translators can identify we have one sentence everywhere and do a sed on their translations.